### DISCUSSION TITLE
Multi-user support?

### DISCUSSION THREAD

**Discussion by andyverbunt** [walt-id/.github#60] [Help] [answered]
I have a question about the supported features.
The documentation talks about "User context separation" and "User data management" but suppose I want to offer a distinct custodial wallet to all the customers of my company. Is that possible?

In other words:
- Is there support multi-user support in wallet-kit and/or ssi-kit? For which components (signatory, custodian, auditor) ?
- if so, where can I find this in the code? It seems the data model does not support this, but of course I might be wrong!
- I did see the use of "tenants" in wallet-kit but it seems this concept is not extended to all components?

**Comment by philpotisk** [comment, marked as answer]
Hi @andyverbunt,

sorry for the delayed answer. The main reason for the delay was that we have relaunched the Web Wallet https://github.com/walt-id/waltid-web-wallet

This is now a white-label solution that should fulfill your requirement "custodial wallet for all customers of your company". Please take a look at the docs + video. Apart of that, you could also drop us a mail if you want to discuss your use case in detail.
